components/button/button.css

```diff
@@ -0,0 +1,110 @@
+.btn {
+  position: relative;
+  display: inline-block;
+  text-align: center;
+  min-width: 150px;
```

Comment:
You can use rem here as you have used it in margin too. Or if you want , then use absolute values everywhere.

Reply:
I use rems for all the margin paddings but I preferred using a fixed value for min-width, max-width, just a choice I made in my library. Really appreciate your review.